src/Aspire.Hosting/Dashboard/DashboardTokenProvider.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I would remove this from this PR and focus solely on the insecure mode. We can do this in a P6 branch.